demos/sev-demo/README.md

```diff
@@ -0,0 +1,192 @@
+Note that viewing the kernel command line from inside the guest via `cat /proc/cmdline` might not accurately reflect the firmware's view of the kernel commandline, which is what the measurement is based on.  
+```
+      "allowed_keys":[
+         "key_id1"
+      ],
+```
+This should correspond to the key in `keys.json`.
+```
+      "allowed-policies":[
+         0
+      ],
+      "min-api-major":0,
+      "min-api-minor":0,
+      "allowed-build-ids":[
+         13
+      ]
+```
+These parameters should be familiar if you have launched any SEV guests before. Note that you can set multiple allowed policies or allowed build IDs for each keyset.
+```
+   }
+}
+```
+
+Once you have the KBS/GOP configuration files setup, simply run the KBS. Note that you can start the KBS with the `-u` flag to skip verification of the measurement. This can be useful for debugging. 
+
+## Preparing an image 
+
+Finally, it's time to prepare an image. You can use the `offline_fs_kbs` sample KBS as a keyprovider. It is compatible with `offline_sev_kbc`. See the [README](https://github.com/confidential-containers/attestation-agent/blob/main/sample_kbs/src/enc_mods/offline_fs_kbs/README.md) for guidance. We will use skopeo (which uses ocicrypt) to encrypt each layer of our image. Rather than specify a fixed encryption key, we specify the KBS as the keyprovider. The KBS will wrap each layer encryption key. These will later be unwrapped by the KBC in the guest. You should end up using a skopeo command that looks something like this. 
+```
+OCICRYPT_KEYPROVIDER_CONFIG=ocicrypt.conf skopeo copy docker://docker.io/<username>/<some-container>:unencrypted docker://docker.io/<username>/<some-container>:encrypted --encryption-key provider:attestation-agent:keys.json:key_id1
+```
+Make sure the sample KBS is running per the linked instructions. You may need to run `skopeo login`. `key_id1` in `keys.json` is exactly the key that needs to be added to `keys.json` in the KBS directory.
+
+## Conclusion
+
+To recap, you will need to make configuration changes to almost every component. Specifically, 
+* OVMF - You need the AMDSEV firmware build.
+* Kernel - You need a kernel with `efi_secret` patches.
+* Initrd - You need the `efi_secret` module, the AA built with `offline_sev_kbc`, and an agent config file with the correct KBC.
+* Shim - You need a PR for the shim and a number of configuration file changes.
+* KBS - You need to provision `keys.json` and `keysets.json` and install the gop-client. 
+* Image - You need to encrypt and image using the sample KBS
+
+Make sure that you have updated the Kata configuration file to point to your new firmware, kernel, and initrd.
```

Comment:
is a snippet example of pointing to each of value here?

Reply:
idk if the conclusion is the place to add this

Reply:
mm true wasn;t thinking in context of conclusion